airbyte-integrations/connectors/source-zendesk-support/source_zendesk_support/source.py

```diff
@@ -104,24 +109,32 @@ def streams(self, config: Mapping[str, Any]) -> List[Stream]:
         :param config: A Mapping of the user input configuration as defined in the connector spec.
         """
         args = self.convert_config2stream_args(config)
-        # sorted in alphabet order
-        return [
-            GroupMemberships(**args),
-            Groups(**args),
-            Macros(**args),
-            Organizations(**args),
-            SatisfactionRatings(**args),
-            SlaPolicies(**args),
-            Tags(**args),
-            TicketAudits(**args),
-            TicketComments(**args),
-            TicketFields(**args),
-            TicketForms(**args),
-            TicketMetrics(**args),
-            TicketMetricEvents(**args),
-            Tickets(**args),
-            Users(**args),
-            Brands(**args),
-            CustomRoles(**args),
-            Schedules(**args),
-        ]
+        all_streams_mapping = {
+            # sorted in alphabet order
+            "group_membership": GroupMemberships(**args),
```

Comment:
could you use stream.name here? (in a dict comprehension for instance)

Reply:
No, because the class instance is not instantiated yet.